src/main/common/bitarray.c

```diff
@@ -36,3 +36,8 @@ void bitArrayClr(void *array, unsigned bit)
 {
     BITARRAY_BIT_OP((uint32_t*)array, bit, &=~);
 }
+
+void bitArrayInv(void *array, unsigned bit)
```

Comment:
Another possibility is : 
```
void bitArrayXor(void* dest, size_t size, void* op1, void* op2)
{
   for(size_t i = 0; i < size; i++)
     ((uint8_t*)dest)[i] = ((uint8_t*)op1)[i] ^ ((uint8_t*)op2)[i];
}
```
and than : 
`bitArrayXor(&newMask, sizeof(newMask), &newMask, &andMask);`

Reply:
I'm down